jsonnet/thanos-mixin/dashboards/receive.libsonnet

```
@@ -0,0 +1,257 @@
+local g = import '../lib/thanos-grafana-builder/builder.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'receive.json':
+      g.dashboard($._config.grafanaThanos.dashboardReceiveTitle)
+      .addTemplate('cluster', 'kube_pod_info', 'cluster', hide=if $._config.showMultiCluster then 0 else 2)
+      .addTemplate('namespace', 'kube_pod_info{%(clusterLabel)s="$cluster"}' % $._config, 'namespace')
+      .addRow(
+        g.row('Incoming Request')
+        .addPanel(
+          g.panel('Rate') +
+          g.httpQpsPanel('thanos_http_requests_total', 'namespace="$namespace",%(thanosReceiveSelector)s' % $._config)
+        )
+        .addPanel(
+          g.panel('Errors') +
+          g.httpErrPanel('thanos_http_requests_total', 'namespace="$namespace",%(thanosReceiveSelector)s' % $._config)
+        )
+        .addPanel(
+          g.panel('Duration') +
+          g.latencyPanel('thanos_http_request_duration_seconds', 'namespace="$namespace",%(thanosReceiveSelector)s' % $._config)
+        )
+      )
+      .addRow(
+        g.row('Detailed')
+        .addPanel(
+          g.panel('Rate') +
+          g.httpQpsPanelDetailed('thanos_http_requests_total', 'namespace="$namespace",%(thanosReceiveSelector)s' % $._config)
+        )
+        .addPanel(
+          g.panel('Errors') +
+          g.httpErrDetailsPanel('thanos_http_requests_total', 'namespace="$namespace",%(thanosReceiveSelector)s' % $._config)
+        )
+        .addPanel(
+          g.panel('Duration') +
+          g.httpLatencyDetailsPanel('thanos_http_request_duration_seconds', 'namespace="$namespace",%(thanosReceiveSelector)s' % $._config)
+        ) +
+        g.collapse
+      )
+      .addRow(
+        g.row('Forward Request')
+        .addPanel(
+          g.panel('Rate') +
+          g.queryPanel(
+            'sum(rate(thanos_receive_forward_requests_total{namespace="$namespace",%(thanosReceiveSelector)s,result="success"}[$interval]))' % $._config,
```

```suggestion
            'sum(rate(thanos_receive_forward_requests_total{namespace="$namespace",%(thanosReceiveSelector)s}[$interval]))' % $._config,
```
We should show the rate of all requests, not only succeeding once, right?